content/en/docs/setup/platform-setup/k3d/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: k3d
+description: Instructions to set up k3d for Istio.
+weight: 30
```

Comment:
The weight needs to place the page alphabetically and should no be the same as another page. the `kind` page uses 30 already.

Reply:
@ericvn fixed 0e2a14c3ed1e2e18084ca60f32ba9f190e3df456